cve_bin_tool/checkers/bolt.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+"""
+CVE checker for bolt
+
+https://www.cvedetails.com/product/64828/Boltcms-Bolt.html?vendor_id=21391
+
+"""
+from cve_bin_tool.checkers import Checker
+
+
+class BoltChecker(Checker):
+    CONTAINS_PATTERNS = [
+        r"State of the ForcePower setting of the bolt daemon.",
+        r"The generation of the Thunderbolt controller associated",
+        r"The maximum generation of any of Thunderbolt controller",
+    ]
+    FILENAME_PATTERNS = [r"bolt"]
+    VERSION_PATTERNS = [
+        r"bolt ([0-9]+\.[0-9]+(\.[0-9])?) starting up.",
```

Comment:
```suggestion
```
Anything that matches the first pattern will also match the second, so we might as well remove the one on line 22 and just keep the one on line 23.

Reply:
Fixed it 👍🏼